### PR TITLE
Deprecate c8s

### PR DESCRIPTION
**Pull request by zmiklank**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,7 +4,6 @@ Redis container image
 [![Build and push images to Quay.io registry](https://github.com/sclorg/redis-container/actions/workflows/build-and-push.yml/badge.svg)](https://github.com/sclorg/redis-container/actions/workflows/build-and-push.yml)
 
 Images available on Quay are:
-* CentOS Stream 8 [redis-6](https://quay.io/repository/sclorg/redis-6-c8s)
 * CentOS Stream 9 [redis-6](https://quay.io/repository/sclorg/redis-6-c9s)
 * Fedora [redis-6](https://quay.io/repository/fedora/redis-6)
```
Comment (phracek) [inline README.md:8]: Redis-5 for FEdora can be also removed. It is not available. The latest is in F36 based on https://koji.fedoraproject.org/koji/packageinfo?packageID=11032

**Review by phracek — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (zmiklank) [reply README.md:8]: I do not see any Redis5 data here. Can you point me to them, please?

Comment (phracek) [reply README.md:8]: Sorry, I have meant redis-6. Sorry for misleading.

**Comment by phracek** [comment]
[test]

**Comment by phracek** [comment]
[test-openshift]

**Pull request merged**